Cargo.toml

```
@@ -12,11 +12,17 @@ edition = "2018"
 [features]
 client = []
 server = []
+
+shared-secret-registration-mac = ["hex", "hmac", "sha1"]
```

To suppress the implicit features generated by optional dependencies by default:

```suggestion
shared-secret-registration-mac = ["dep:hex", "dep:hmac", "dep:sha1"]
```